### PR TITLE
DDF-4106 Updates Admin UI to use webpack

### PR DESCRIPTION
**Pull request by andrewkfiedler**
Note that I still need to remove some now unused things, like the grunt config.  Hoping to get some eyes on this and maybe some testing.  As far as I can tell, everything works.

#### What does this PR do?
 - Updates Admin UI to start using webpack

#### Who is reviewing it? 
@tbatie
@djblue 
@adimka 
@Bdthomson

### CODE REVIEW

**Comment by andrewkfiedler** [comment]
build now

File: ui/packages/ui/package.json [hunk 1]
```diff
@@ -47,17 +47,43 @@
...
+    "path": "0.4.9",
+    "backbone-relational": "0.8.8",
+    "iframe-resizer": "3.6.3",
+    "js-cookie": "2.2.0",
+    "backbone.marionette": "1.8.8",
+    "backbone.modelbinder": "1.1.0",
+    "backbone-poller": "1.1.3",
+    "backbone-associations": "0.6.2",
+    "blueimp-file-upload": "9.25.1",
+    "bootstrap-multiselect": "0.9.13-1",
```
Comment (djblue) [inline ui/packages/ui/package.json]: I think the `version-validation-plugin` didn't like this line

**Comment by djblue** [comment]
I think we can get rid of some of the devDependencies.

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by djblue** [comment]
### Hero Success 🎉 

- went through installer
- edited configuration
- toggled features
- ensured documentation still shows up
- checked dev server still works
- check admin ui extension
  - map layers
  - sources
  - log viewer
  - default layout
  - certificates

**Comment by peterhuffer** [comment]
Successful hero on downstream.

**Pull request merged**